packages/clay-empty-state/src/EmptyState.tsx

```diff
@@ -0,0 +1,67 @@
+/**
+ * © 2020 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import ClayButton from '@clayui/button';
+import React from 'react';
+
+const emptyStateImg = require('../images/empty_state.gif');
+const searchStateImg = require('../images/search_state.gif');
+const successStateImg = require('../images/success_state.gif');
+
+export const DISPLAY_STATES = {
+	EMPTY: 'empty',
+	SEARCH: 'search',
+	SUCCESS: 'success',
+};
+
+interface IProps extends React.HTMLAttributes<HTMLDivElement> {
+	actionLabel: string;
+	description: string;
+	displayState: string;
+	onClickAction: () => void;
+	title: string;
+}
+
+const ClayEmptyState: React.FunctionComponent<IProps> = ({
+	actionLabel,
+	description,
+	displayState,
+	onClickAction,
+	title,
+}) => {
+	let imageState;
+
+	if (displayState === DISPLAY_STATES.EMPTY) {
+		imageState = emptyStateImg;
+	} else if (displayState === DISPLAY_STATES.SEARCH) {
+		imageState = searchStateImg;
+	} else if (displayState === DISPLAY_STATES.SUCCESS) {
+		imageState = successStateImg;
+	}
+
+	return (
+		<div
+			className={`empty-state-root ${!imageState &&
+				'empty-state-root--no-image'}`}
+		>
+			{imageState && <img alt="empty-state-image" src={imageState} />}
+
+			<h2>{title || 'No results found'}</h2>
```

Comment:
Rather than put the `||` here, we typically just set a default value to the prop. `title = 'No results found',`